powerdnsadmin/models/user.py

```diff
@@ -307,7 +307,20 @@ def is_validate(self, method, src_ip='', trust_user=False):
                                                 LDAP_USER_GROUP))
                                     return False
                             elif LDAP_TYPE == 'ad':
-                                user_ldap_groups = []
+                                if LDAP_ADMIN_GROUP:
```

Comment:
Can you initialise all the filters with an empty string and just use a if ?

```suggestion
                                ldap_admin_group_filter, ldap_operator_group, ldap_user_group = "", "", ""
                                if LDAP_ADMIN_GROUP:
                                    ldap_admin_group_filter = "(memberOf:1.2.840.113556.1.4.1941:={0})".format(LDAP_ADMIN_GROUP)
```
Therefore there are many words cut by spaces (maybe your editor) in the PR to fix.

Reply:
hello @jbe-dw 
Thanks for the feedback. I removed the unused method and replaced the loop syntax. Please let me know if you find something else I have missed.